src/hub/libraries/AssetLogic.sol

```diff
@@ -80,11 +80,21 @@ library AssetLogic {
   /// @notice Returns the total added assets for the specified asset.
   function totalAddedAssets(IHub.Asset storage asset) internal view returns (uint256) {
     uint256 drawnIndex = asset.getDrawnIndex();
+
+    uint256 premiumRay = Premium.calculatePremiumRay({
+      premiumShares: asset.premiumShares,
+      drawnIndex: drawnIndex,
+      premiumOffsetRay: asset.premiumOffsetRay,
+      realizedPremiumRay: asset.realizedPremiumRay
+    });
+    uint256 aggregatedOwedRay = (uint256(asset.drawnShares) * drawnIndex) +
```

Comment:
is it worth creating a function to get aggregatedOwedRay? we can use it here and twice in getUnrealizedFees, wdyt?